### PR TITLE
fix(gce-node): disable sshguard on gce ubuntu bionic/xenial

### PR DESCRIPTION
**Pull request by amoskong**
(cherry picked from commit a7a3d5345b6a9ce051608aa101cb85f9cebb1df1)

(manually backport part of commit 657d8ddf14c0e499bb2c880e8ec3892d79e3429e)
Added extra_meta parameter for gce_create_metadata function.

Signed-off-by: Amos Kong <amos@scylladb.com>

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by amoskong** [comment]
Testing this PR by: https://jenkins.scylladb.com/job/scylla-staging/job/amos/job/rolling-upgrade-2019.1/job/rolling-upgrade-ubuntu16.04/5/

File: sdcm/utils/common.py [hunk 1]
```diff
@@ -984,7 +984,7 @@ def tag_ami(ami_id, tags_dict, region_name):
     LOGGER.info("tagged %s with %s", ami_id, tags)
 
 
-def get_non_system_ks_cf_list(loader_node, db_node, request_timeout=300, filter_out_table_with_counter=False,
+def get_non_system_ks_cf_list(loader_node=None, db_node=None, request_timeout=300, filter_out_table_with_counter=False,
```
Comment (bentsi) [inline sdcm/utils/common.py:987]: how this is related?

Comment (bentsi) [reply sdcm/utils/common.py:987]: @amoskong ?

Comment (amoskong) [reply sdcm/utils/common.py:987]: This fixed pylint error.

**Pull request merged**